pcdsdevices/epics_motor.py

```diff
@@ -753,7 +755,8 @@ def find_configuration(pattern, case_insensitive=True, display=20):
         Returns a list of strings if display is None, and nothing otherwise.
         """
         IMS._setup_and_check_pmgr()
-        matches = IMS._pm.match_config(pattern, ci=case_insensitive)
+        # matches = IMS._pm.match_config(pattern, ci=case_insensitive)
+        matches = IMS._pm.match_config(pattern, ci=case_insensitive, parent='USR')
```

Comment:
I think this update needs to be added to the PR description + the pre-release notes: "Make IMS pmgr only search through USR objects" I think you implemented this one a long time ago and it never made it into master branch.